README.md

```diff
@@ -73,55 +154,97 @@ USE_L10N = True
 # as found in the Transifex UI under your project
 TRANSIFEX_TOKEN = <project_token>  # used for pulling translations from Transifex
 TRANSIFEX_SECRET = <project_secret>  # used for pushing source content to Transifex
-TRANSIFEX_SYNC_INTERVAL = <seconds> # used for defining the daemon running interval in seconds
 ```
 
 A list of supported language codes is available [here](https://www.transifex.com/explore/languages/) and should
 be declared in the `ll-cc` format, compatible with the `Accept-Language` HTTP header specification, for example
 `pt-br` instead of `pt_BR`.
 
-### Quick guide
+#### Quickstart
 
 These are the minimum steps required for testing the Transifex Toolkit with a Django project end-to-end:
 
-1. Add translation hooks in your templates
-2. Push the source content to Transifex
-3. Translate content on Transifex
-4. Display translated content to your users
+1. Install Transifex Python toolkit in your Django project, using the pip install transifex-python in your console. Read more about [toolkit installation](#installation).
+2. Update your Django settings file with instruction on using the Transifecx Native toolkit. Read more about [configuring your project](#configuration).
```

Comment:
2. Update your **project's Django settings file with the settings required for using the Transife~~c~~x Native toolkit**

Reply:
Done